src/components/box/box.css

```diff
@@ -0,0 +1,2 @@
+.box {
```

Comment:
What does this do?

Reply:
it was a copy directly from gui. I can remove it – I thought it might be good to mirror gui more directly, but that seems unnecessary with this